app/models/invenio_rdm_record_converter.rb

```diff
@@ -508,4 +515,19 @@ def funding(file_id)
     # remove nil values
     funding_sources.compact
   end
+
+
+  def map_collections_to_communities(communities=@communities)
```

Comment:
I added communities as an argument to help with testing this out. `@@collections_to_communities` is just the result of reading JSON.

Reply:
This has been updated. Marking as resolved for my own tracking.